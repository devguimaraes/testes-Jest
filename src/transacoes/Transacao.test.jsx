import { render } from "@testing-library/react";
import React from "react";
import Transacao from "./Transacao";

describe("Componente de transacao do extrato", () => {
  test("Snapshot do componente deve permancer sempre o mesmo", async () => {
    const { container } = await render(
      <Transacao data="02/11/2022" tipo="saque" valor="20.00" />
    );

    expect(container).toMatchSnapshot();
  });
});
