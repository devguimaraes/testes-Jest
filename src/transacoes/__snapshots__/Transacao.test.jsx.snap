// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transacao do extrato Snapshot do componente deve permancer sempre o mesmo 1`] = `
<div>
  <div
    class="transacao-container"
  >
    <p>
      02/11/2022
    </p>
    <p>
      saque
    </p>
    <p>
      R$ 
      20.00
    </p>
  </div>
</div>
`;
